src/FindTranslocations.cpp

```diff
@@ -79,119 +199,128 @@ int main(int argc, char *argv[]) {
 	}
 	alignmentFile = vm["bam"].as<string>();
 
-	if (!vm.count("max-insert")) {
-		DEFAULT_CHANNEL << "Please specify max-insert " << endl;
-		return 2;
+	if (vm.count("output")) {
+		string header = vm["output"].as<string>();
+		outputFileHeader = header ;
 	}
 
-	if (vm.count("min-insert")) {
-		min_insert = vm["min-insert"].as<int>();
-		if(min_insert <= 0) {
-			DEFAULT_CHANNEL << "minimum insert should be at least 1\n";
-			DEFAULT_CHANNEL << desc << endl;
-			return 2;
-		}
-	}
 
-	if (vm.count("max-insert")) {
-		max_insert = vm["max-insert"].as<int>();
+	if(vm.count("extract")){
```

Comment:
Maybe the code needs some reorganisation... the program might have run modalities. --extract extracts from the bam file regions, --struct-var looks for structural variations, --copy-number looks for copy number variations, and the modalities are exclusive, i.e., you can run the program only in one of the modalities (like it happens for samtools)